sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -267,6 +385,24 @@ public ForceUnlinkRedisEnterpriseDatabaseContent(System.Collections.Generic.IEnu
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.RedisEnterprise.Models.ForceUnlinkRedisEnterpriseDatabaseContent>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.RedisEnterprise.Models.ForceUnlinkRedisEnterpriseDatabaseContent>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct HighAvailability : System.IEquatable<Azure.ResourceManager.RedisEnterprise.Models.HighAvailability>
```

Comment:
The current name of this struct is too generic. Please rename it to something more descriptive.

Reply:
what will be the effect on changing the name here ?